src/ClientSideMonitoring/AbstractMonitoringMiddleware.php

```diff
@@ -224,6 +224,23 @@ protected function populateResultEventData(
         return $event;
     }
 
+    
+    /**
+     * Checks if the socket is created.
+     *
+     * @return bool Returns true if the socket is created, false otherwise.
+     */
+    private function isSocketCreated()
+    {
+        // Before version 8, sockets are resources
+        // After version 8, sockets are instances of Socket
+        if (PHP_MAJOR_VERSION >= 8) {
+                !(self::$socket instanceof Socket);
```

Comment:
Do you mean to do a return statement as following:
```
return self::$socket instanceof \Socket;
```
If so, could you please make that adjustment?

Thanks.

Reply:
Yes, sorry about that - not sure how I missed that - I've added the return statement and properly referenced the PHP Socket class